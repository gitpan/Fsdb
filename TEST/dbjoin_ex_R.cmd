prog='dbjoin'
args='-S --input TEST/dbjoin_ex_R.in-2 -n cid'
cmp='diff -cb '
