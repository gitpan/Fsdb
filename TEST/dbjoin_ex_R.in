#fsdb -R C sid cid
sid: 1
cid: 10

sid: 2
cid: 11

sid: 1
cid: 12

sid: 2
cid: 12

#  | dbsort -n cid
#   | dbfilealter -R C
