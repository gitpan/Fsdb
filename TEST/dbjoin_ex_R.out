#fsdb -R C cid sid cname
cid: 10
sid: 1
cname: pascal

cid: 11
sid: 2
cname: numanal

cid: 12
sid: 1
cname: os

cid: 12
sid: 2
cname: os

#   | dbfilealter -R C
#  | dbsort -n cid
#   | dbfilealter -R C
#   | dbjoin -S --input TEST/dbjoin_ex_R.in-2 -n cid
